src/rpc/core_rpc_server_commands_defs.h

```diff
@@ -1673,7 +1673,27 @@ namespace cryptonote
         KV_SERIALIZE(out_peers)
       END_KV_SERIALIZE_MAP()
     };
-    
+
+    struct response
+    {
+	  std::string status;
+
+      BEGIN_KV_SERIALIZE_MAP()
+        KV_SERIALIZE(status)
+      END_KV_SERIALIZE_MAP()
+    };
+  };
+
+  struct COMMAND_RPC_IN_PEERS
+  {
+	struct request
+    {
+	  uint64_t in_peers;
+      BEGIN_KV_SERIALIZE_MAP()
```

Comment:
Would be nice to get proper indent on new code.

Reply:
Done.